build-packages/magento-scripts-php-extensions/extensions/sourceguardian.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path')
+
+const archivesByArch = {
+    x64: 'https://www.sourceguardian.com/loaders/download/loaders.linux-x86_64.tar.gz',
+    arm64: 'https://www.sourceguardian.com/loaders/download/loaders.macosx-arm64.tar.gz'
```

Comment:
arm64 version should use loaders for arm64 Linux since extension will be installed in Linux container environment.